### ISSUE TITLE
[UMP] Exception raised when calling Update of ConsentInformation on Android

### ISSUE THREAD

**Issue by kevinchailloux** [googleads/googleads-mobile-unity#2611]
### [REQUIRED] Step 1: Describe your environment

  * Unity version: 2021.3.21
  * Google Mobile Ads Unity plugin version: 8.0.0
  * Platform: Android
  * Platform OS version: Android 9 & 12
  * Any specific devices issue occurs on: Pixel 3a XL
  * Mediation ad networks used, and their versions: None

#### Steps to reproduce:

- Initialize the Mobile Ads SDK (or not, tried both)
- Call ConsentInformation.Update (tried with empty request of with argumented request)

#### Relevant Code:

```
            MobileAds.RaiseAdEventsOnUnityMainThread = true;
            MobileAds.Initialize(status => {
                ConsentRequestParameters consentRequestParameters = new ConsentRequestParameters();
                ConsentInformation.Update(consentRequestParameters, OnConsentInfoUpdated);
            });
```

The generated error is 
```
NullReferenceException: Object reference not set to an instance of an object.
      at GoogleMobileAds.Ump.Android.Utils.GetConsentRequestParametersJavaObject (GoogleMobileAds.Ump.Api.ConsentRequestParameters consentRequestParameters, UnityEngine.AndroidJavaObject activity) [0x00000] in <00000000000000000000000000000000>:0 
      at GoogleMobileAds.Ump.Android.ConsentInformationClient.Update (GoogleMobileAds.Ump.Api.ConsentRequestParameters request, System.Action onConsentInfoUpdateSuccessCallback, System.Action`1[T] onConsentInfoUpdateFailureCallback) [0x00000] in <00000000000000000000000000000000>:0 
      at UnityEngine.UnitySynchronizationContext+WorkRequest.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
      at UnityEngine.UnitySynchronizationContext.Exec () [0x00000] in <00000000000000000000000000000000>:0 
```

#### Additional informations
My Google Mobile Ads Settings is filled only with admob app ids.

This is the config of the Mobile Ads SDK

**Comment by NVentimiglia** [comment]
Hi @kevinchailloux 

I have confirmed the issue. Thank you for the report. We will fix this issue.

**Comment by NVentimiglia** [comment]
This should be resolved in the latest release.

**Comment by snappedToGrid** [comment]
I'm working with v8.3.0 and this doesn't work. I had to use [UnityMainThreadDispatcher] in the callbacks in order to force the main thread, even after explicitly turning on the "MobileAds.RaiseAdEventsOnUnityMainThread = true;"  command before making any UMP calls. 

I had to spend a lot of time on this because Unity was refusing to load scenes and assets without giving me any errors that I'm trying to do that from the background thread, and I kind of had to guess, that despite setting this to true, it does not run on the main thread.

(https://github.com/PimDeWitte/UnityMainThreadDispatcher)
